lib/action_view/component/base.rb

```diff
@@ -169,13 +169,19 @@ def identifier
 
         private
 
+        def matching_views_in_source_location
```

Comment:
If we follow this comment https://github.com/github/actionview-component/pull/123/files#r349892608 we may not need to extract this into a method right? Cause it seems like it was extracted just to be able to stub it in the test no?